dashboard/config/routes.rb

```diff
@@ -79,12 +79,24 @@ module OPS
   get '/u/:id', to: redirect('/c/%{id}')
   get '/u/:id/:action_id', to: redirect('/c/%{id}/%{action_id}')
 
+  # These links should no longer be created (August 2017), though we will continue to support
+  # existing links. Instead, create /r/ links.
```

Comment:
/r/ links won't work if a user is deleted, and yes there are work arounds, but I think they were decided to be good enough security. 
/c/ links will still work if you do as directed.
All of this is correct and expected, but again, correct me if I'm wrong Asher.